test/unit/helpers/bpmn-model-utils.ts

```diff
@@ -157,7 +155,7 @@ interface BaseElement {
   name?: string;
 }
 
-type Pool = ContainerWithLanes & BaseElement;
+type Pool = ContainerWithLanes & BaseElement & { hidden?: boolean };
```

Comment:
⚠️ we are using the word 'hidden' here whereas we are using 'not displayed' everywhere else. This is not consistent
See also https://github.com/process-analytics/bpmn-visualization-js/pull/2129#discussion_r930016802